src/github/entities/user-installation.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserInstallation extends Base {
+  @Column({ type: 'varchar', length: 20 })
+  installationId: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  organizationName: string;
+
+  @ManyToOne(() => User, (user) => user.installations)
```

Comment:
PullProofAI

Consider adding a cascade option to the @ManyToOne relationship to handle cascading operations like deletion of related entities.
```diff
+   @ManyToOne(() => User, (user) => user.installations, { cascade: true })
```


<!-- This is an auto-generated comment by PullProofAI -->